--- unifi-video.orig	2019-05-30 10:04:45.000000000 -0700
+++ unifi-video	2019-05-30 10:12:01.776834161 -0700
@@ -277,7 +277,7 @@
 		 -Dcom.sun.management.jmxremote.authenticate=false \
 		 -Dcom.sun.management.jmxremote.port=${JVM_JMXREMOTE_PORT}"
 	[ -z "${JVM_JMXREMOTE_HOST}" ] && \
-		JVM_JMXREMOTE_HOST=$(hostname -I | cut -d' ' -f1)
+		JVM_JMXREMOTE_HOST=$(hostname -i | cut -d' ' -f1)
 	[ -z "${JVM_JMXREMOTE_HOST}" ] || \
 		JVM_OPTS="${JVM_OPTS} -Djava.rmi.server.hostname=${JVM_JMXREMOTE_HOST}"
 
@@ -337,7 +337,7 @@
 	start)
 	require_root
     update_limits
-        ulimit -H -c 200
+        #ulimit -H -c 200
         echo 0x10 > /proc/self/coredump_filter
 		if is_service_running "${PIDFILE}" >/dev/null; then
 			log_verbose "${NAME} is already running..."
